engine/src/main/resources/application-base.yml

```diff
@@ -111,4 +111,14 @@ callback:
       backoff-policy:
         initial-interval-ms: 5000
         max-interval-ms: 30000
-        multiplier: 2.0
\ No newline at end of file
+        multiplier: 2.0
+
+docker:
+  registry:
+    url: https://registry-1.docker.io
+    username:
+    password:
+use:
```

Comment:
"use" is a weird prefix. I suggest to put all the docker configuration under 'docker' prefix. e.g.:
`use.docker` -> `docker.enabled`
`use.socket.path` -> `docker.socket`